python/cuml/test/test_random_forest.py

```diff
@@ -1139,7 +1156,16 @@ def test_rf_nbins_small(small_clf):
     # Initialize, fit and predict using cuML's
     # random forest classification model
     cuml_model = curfc()
-    cuml_model.fit(X_train[0:3, :], y_train[0:3])
+
+    # display warning when nbins less than samples
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter("always")
+        cuml_model.fit(X_train[0:3, :], y_train[0:3])
+        assert("The number of bins, `n_bins` is greater than "
+               "the number of samples used for training. "
+               "Changing `n_bins` to number of training samples."
+               in str(w[-1].message))
+        print(str(w[-1].message))
```

Comment:
```suggestion
```

I don't think it is necessary to print the message, maybe only if it is wrong?

Reply:
oh yea, that's on me will get rid of it, dante